virtcontainers/qemu.go

```diff
@@ -21,6 +17,11 @@ import (
 	"time"
 	"unsafe"
 
+	govmmQemu "github.com/intel/govmm/qemu"
```

Comment:
You probably should have put the import re-order in a separate commit ;-), then I'd not be typing in this little box :-)